rockcraft/commands/init.py

```diff
@@ -116,6 +117,30 @@ class InitCommand(AppCommand):
                 #       - flask/app/webapp
                 #       - flask/app/templates
                 #       - flask/app/static
+                #       # You may need packages to build a python package. Add it here if necessary
+                #       build-packages:
+                #           - pkg-config
+                #           - libxmlsec1-dev
+
+                # # You can add package slices or Debian packages to the image.
+                # # Package slices are subsets of Debian packages, which result
+                # # in smaller and more secure images.
+                # # See https://documentation.ubuntu.com/rockcraft/en/latest/explanation/chisel/
```

Comment:
Double comment?

Reply:
It was before (it is also in the django profile), I understand that is was put there so the user can remove the first line of comments with his/her editor and still have functional yaml. Do you think it would be better to remove it?